extensions/resource-deployment/src/services/azdataService.ts

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as path from 'path';
+import { IPlatformService } from './platformService';
+import { BigDataClusterDeploymentProfile } from './bigDataClusterDeploymentProfile';
+
+interface BdcConfigListOutput {
+	stdout: string[];
+}
+
+export interface IAzdataService {
+	getDeploymentProfiles(): Thenable<BigDataClusterDeploymentProfile[]>;
+}
+
+export class AzdataService implements IAzdataService {
+	constructor(private platformService: IPlatformService) {
+	}
+
+	getDeploymentProfiles(): Thenable<BigDataClusterDeploymentProfile[]> {
+		return this.ensureWorkingDirectoryExists().then(() => {
+			return this.getDeploymentProfileNames();
+		}).then((names: string[]) => {
+			return Promise.all(names.map(name => { return this.getDeploymentProfileInfo(name); }));
+		});
+	}
+
+	private async getDeploymentProfileNames(): Promise<string[]> {
+		const env: NodeJS.ProcessEnv = {};
+		// azdata requires this environment variables to be set
+		env['ACCEPT_EULA'] = 'yes';
+		const cmd = 'azdata bdc config list -o json';
+		// Run the command twice to workaround the issue:
+		// First time use of the azdata will have extra EULA related string in the output
+		// there is no easy and reliable way to filter out the profile names from it.
```

Comment:
This is even with ACCEPT_EULA being set? Have you filed a bug?

Reply:
yes, i did, Brandon will fix it.